master/buildbot/status/status_gerrit.py

```diff
@@ -156,6 +157,13 @@ def __init__(self, server, username, reviewCB=DEFAULT_REVIEW,
         self.gerrit_version = None
         self.gerrit_version_time = 0
         self.gerrit_identity_file = identity_file
+        notify_values = ['NONE', 'OWNER', 'OWNER_REVIEWERS', 'ALL']
+        if notify in notify_values or notify is None:
```

Comment:
[nit] I'd re-order the conditions: `None` case is more common and this comparison is cheaper.